src/NzbDrone.Core/Tv/SeriesEditedService.cs

```diff
@@ -1,24 +1,23 @@
-﻿using NzbDrone.Core.Messaging.Commands;
+﻿using NzbDrone.Core.MediaFiles;
 using NzbDrone.Core.Messaging.Events;
-using NzbDrone.Core.Tv.Commands;
 using NzbDrone.Core.Tv.Events;
 
 namespace NzbDrone.Core.Tv
 {
     public class SeriesEditedService : IHandle<SeriesEditedEvent>
     {
-        private readonly IManageCommandQueue _commandQueueManager;
+        private readonly IDiskScanService _diskScanService;
 
-        public SeriesEditedService(IManageCommandQueue commandQueueManager)
+        public SeriesEditedService(IDiskScanService diskScanService)
         {
-            _commandQueueManager = commandQueueManager;
+            _diskScanService = diskScanService;
         }
 
         public void Handle(SeriesEditedEvent message)
         {
             if (message.Series.SeriesType != message.OldSeries.SeriesType)
             {
-                _commandQueueManager.Push(new RefreshSeriesCommand(message.Series.Id));
+                _diskScanService.Scan(message.Series);
```

Comment:
https://github.com/Sonarr/Sonarr/commit/25abeb8c9c267218f66e449fff9eab377615af42 - I specifically called out rescanning in the commit message, but that was before the special refresh logic, but when we were adding absolute episode numbers ourselves, but only for anime: https://github.com/Sonarr/Sonarr/blob/25abeb8c9c267218f66e449fff9eab377615af42/src/NzbDrone.Core/Tv/RefreshEpisodeService.cs#L156

The only really benefit I see with a refresh/rescan after editing now is for picking up air-by-date episodes when switch to daily.